css/style.css

```diff
@@ -41,4 +35,277 @@ body{
+  display: inline-block;
+  vertical-align: middle;
+  margin-right: 40px;
+  outline: 2px dotted blue;
+}
+
+ul,
+h1,
+h2,
+body{
+  outline: 2px dotted grey;
+  margin: 0;
+  padding: 0;
+}
+
+
+
+.main-navigation{
+  width: 780px;
+  float: left;
+}
+
+.user-block{
+  width: 140px;
+  float: right;
+  height: 32px;
+  outline: 2px dotted yellow;
+}
+
+.clearfix::after{
+  content: "";
+  display: block;
+  clear: both;
+}
+
+.index-logo{
+  width: 368px;
+  height: 204px;
+  margin-right: auto;
+  margin-left: auto;
+  margin-bottom: 25px;
+  outline: 2px dotted red;
+}
+
+.features-item{
+  float: left;
+  width: 300px;
+  margin-right: 20px;
+  outline: 2px dotted red;
+  min-height: 170px;
+}
+
+.features-item:last-child{
+  margin-right: 0;
+}
+
+.features{
+  margin-bottom: 80px;
+}
+
+.index-content{
+  padding: 50px 80px;
+  margin-bottom: 35px;
+  outline: 2px dotted green;
+}
+
+.index-content:last-child{
+  margin-bottom: 65px;
+}
+
+.index-content-left{
+  float: left;
+  width: 380px;
+  min-height: 320px;
+  outline: 2px dotted red;
+}
+
+.index-content-right{
+  float: right;
+  width: 300px;
+  min-height: 320px;
+  outline: 2px dotted red;
+}
+
+.index-content:last-child .index-content-left,
+.index-content:last-child .index-content-right{
+  min-height: 285px;
+}
+
+.main-footer{
+  outline: 2px dotted red;
+  padding-top: 60px;
+  padding-bottom: 40px;
+  background-image: url(../img/background-dark.gif);
+}
+
+.footer-contacts{
+  float: left;
+  width: 320px;
+  outline: 2px dotted red;
+  margin-right: 60px;
+  min-height: 90px;
+}
+
+.footer-social{
+  float:left;
+  width: 180px;
+  outline: 2px dotted red;
+  min-height: 90px;
+}
+
+.footer-copyright{
+  float: right;
+  width: 180px;
+  outline: 2px dotted red;
+  min-height: 90px;
+}
+
+.other-header{
+  background-color: #000000;
+  padding: 25px 0;
+  outline: 2px dotted blue;
+}
+
+.main-navigation-shop{
+  width: 780px;
+  float: left;
+  outline: 2px dotted red;
+}
+
+.catalog-content{
+  padding-top: 60px;
+  margin-bottom: 30px;
+}
+
+.product-title{
+  min-height: 75px;
+  outline: 2px dotted blue;
+  width: 940px;
+  float: left;
+  margin-bottom: 55px;
+}
+
+.filters{
+  outline: 2px dotted red;
+  float: left;
+  min-height: 680px;
+  width: 240px;
+}
+
+.manufacturers-block{
+  float: left;
+  min-height: 310px;
+}
+
+.filter-name{
+  float: left;
+  min-height: 55px;
+  width: 240px;
+  outline: 2px dotted red;
+}
+
+.filter-item{
+  float: left;
+  width: 240px;
+  outline: 2px dotted red;
+}
+
+.filter-name:last-child{
+  margin-top: 55px;
+}
+
+.filter-button{
+  float: left;
+  width: 240px;
+  outline: 2px dotted red;
+}
+
+.products{
+  outline: 2px dotted red;
+  float: left;
+  min-height: 680px;
+  width: 700px;
+}
+
+.product-card{
+  float: left;
+  width: 220px;
+  min-height: 290px;
+  outline: 2px dotted red;
+  margin-right: 20px;
+  margin-bottom: 20px;
+}
+
+.product-card:nth-child(3),
+.product-card:nth-child(6){
```

Comment:
You can use "nth-child(3n)" to choose every third element. See example: https://www.w3schools.com/cssref/tryit.asp?filename=trycss3_nth-child_formula